src/redux/reducers/fetchEpisodes.js

```diff
@@ -7,4 +7,4 @@ export default (state = {}, action) => {
    default:
     return state
   }
- }
\ No newline at end of file
+ }
```

Comment:
If I can suggest to plan store structure - name of file will be also name of key in the store, that's why 'episodes'. Please re-think `result` name of key. Your structure look like that now: 
```
store = {
   fetchEpisodes: {
      result: {payloadGoesHere}
   }
}
```